src/ucp/core/ucp_ep.c

```diff
@@ -780,6 +774,11 @@ ucp_ep_create_api_conn_request(ucp_worker_h worker,
     ucp_ep_h           ep;
     ucs_status_t       status;
 
+    if (params->field_mask & UCP_EP_PARAM_FIELD_CLIENT_ID) {
+        ucs_error("client id is supported only for sockaddr connection establishment");
```

Comment:
does this mean that client id is not supported for RDMACM?

Reply:
No, this ment to set client_id only for ep creation by sockaddr, neither by worker_addr nor by conn_request